Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoIngestModule.java

```diff
@@ -356,7 +363,8 @@ private AbstractFile getAbstractFile(String file) {
                 return abstractFiles.get(0);
             }
             for (AbstractFile resolvedFile : abstractFiles) {
-                if (filePath.matches(resolvedFile.getParentPath().toLowerCase())) {
+                // double check its an exact match
+                if (filePath.toLowerCase().matches(resolvedFile.getParentPath().toLowerCase())) {
```

Comment:
why are we even using matches()?  there is no regex invlolved.  we could just use equalsIgnoreCase()

Reply:
Done.